FutureNorthants/northants-website
# Added news_feed graphql query to backend

### Testing
1. With this branch checked out, restart your backend.
2. Visit https://core-cms.northants.localhost/graphql and enter the following query: `{ news_feed { title, description, copyright, language, updated, entries { id, title, link, content, author, image, date } } }` and verify that sensible results are returned with values in all fields
3. Enter the following query: `{ news_feed_latest { date } }` and verify it returns the date of the most recent published news article in the CMS.
4. You could opt to try other known queries to ensure they haven't been broken (as this is purely a new query with no changes to others, this is unlikely).
